packages/web-react/src/components/FileUploader/FileUploaderAttachment.tsx

```diff
@@ -21,6 +41,12 @@ const FileUploaderAttachment = (props: SpiritFileUploaderAttachmentProps) => {
     }
   };
 
+  const editHandler = () => null;
```

Comment:
I think we should remove this. The handler should be passed through props as `onEdit/onModify/onCustomAction/onSomethingElse`. By using this handler from prop the `isEditable` prop is useless and can be replaced by the handler, e.g.: `isEditable && <...` -> `onEdit && <...`.